packages/uninstaller/src/aws.ts

```diff
@@ -177,14 +177,25 @@ export function* destroyAWSInfra(): Generator<
 
   const taskGroup4 = [];
 
-  const route53dnsname = `${destroyConfig.clusterName}.opstrace.io.`;
-  taskGroup4.push(yield fork(route53PurgeZonesForDnsName, route53dnsname));
-  taskGroup4.push(
-    yield fork(
-      [opstraceClient, opstraceClient.delete],
-      destroyConfig.clusterName
-    )
-  );
+  // Note(JP): do not delete any DNS-related infra when the Opstrace instance
+  // was set up with a custom DNS name. TODO: when the Opstrace instance
+  // configuration document is not input for the destroy operation, we cannot
+  // 'know' that this instance was configured to use a custom DNS name. Can we
+  // peek into the cluster?
```

Comment:
Idea: probe for existence of `<instance_name>.opstrace.io` and if it does not exist: skip DNS teardown. Maybe require N DNS name resolution failures to reduce the likelihood for false negatives. Two disadvantages:
- false negatives: when it appears to not exist, but exists: erroneously skipping teardown. Fallout: orphaned entry in Opstrace DNS service for the acting identity. That's fine for said identity, but blocks an instance_name from being reused by other DNS service users.
- false positives: when it exists but belongs to a different org/identity -- then, we'd perform the DNS Service login and find that we cannot delete said entry -- in this case, the `destroy` command should not crash.

Thinking about this, I think it's safe to always call
```
taskGroup4.push(yield fork(route53PurgeZonesForDnsName, route53dnsname));
```
because this will do no harm in the context of a user's AWS account (and finishes quickly).